code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -79,6 +79,8 @@
 
 /obj/machinery/smartfridge/attackby(obj/item/O, mob/user, params)
 	if(default_deconstruction_screwdriver(user, "smartfridge_open", "smartfridge", O))
+		if(istype(src, /obj/machinery/smartfridge/drying_rack)) //See if it's a drying rack because the screwdrivers screw it up hehe geddit
```

Comment:
@Funce you can make your own proc called 
/obj/machinery/smartfridge/drying_rack/attackby